gui/gui.py

```diff
@@ -48,6 +48,45 @@ def get_control_mode(window):
 
     file.close()
 
+def manual_mode_actions(window, event, curr_control_mode):
```

Comment:
honestly shouldnt include changes in this file in this pr

Reply:
oof sorry github didn't send me an email thought it would've. @AC408 could you add this to a separate PR as you mentioned? It is pretty separate from the changes in this PR and I think it would benefit to test this independently (+ bc potential merge conflicts w/ the rpi to GUI pr). In general tho the code for this GUI file seems good to me!